multiterm/src/Makefile.am

```diff
@@ -24,6 +24,8 @@ multiterm_la_SOURCES = \
 	terminal.vala
 
 multiterm_la_CFLAGS = \
+	-DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' \
+	-DLOCALEDIR='"$(localedir)"' \
```

Comment:
I think I like that way better, and IIRC I've done it before on some project. If you feel like doing it go ahead, otherwise this way is fine too, at least the globals are constant.

Would it make more sense to have these in the Geany .vapi?